index.html

```diff
@@ -1412,6 +1412,123 @@ <h3>Biometrics</h3>
+    <h2>Environmental and Ethical Considerations</h2>
+
+    <p>
+      The following section details certain key areas of focus when implementing a 
+      new DID menthod, or implementing a solution that incorporates DIDs.
+    </p>
+
+    <p>
+      A reliable guide for making assessments of various technologies and weighing
+      ethical considerations is the 
+      <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG Ethical Web Principles</a>
+      document.
+    </p>
+
+    <section class="informative">
+      <h3>Broad Ethical Principles</h3>
+      <p>
+        As with all things, strong consideration for the appropriate and ethical use
+        of technology should be made when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">Principles</a>
+        section of the Ethical Web Principles, there are certain key goals that should 
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Energy Usage and Environmental Impacts</h3>
+      <p>
+        When implementing or utilizing a DID method, consideration should be given
+        to the environmental impacts of any underlying technologies.  
+        Avoidance of excess energy use that does not further other equally important 
+        goals such as human rights or personal privacy, particularly when that energy 
+        is sourced from non-sustainable energy creation methods, should be a best 
+        practice that serves the interest of all.
+      </p>
+      <p>
+        Direct comparison of each factor, and the balances between them, should be
+        considered when evaluating particular methods or approaches. Notably, the
+        ability to misuse or control DIDs that could impede privacy might be
+        found to be inversely correlated with additional energy consumption.
+      </p>
+      <p>
+        The guiding principle that 
+        <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+        web must be an environmentally sustainable platform</a> should be followed.
+      </p>
+
+      <p class="advisement">
+        Utilizing or authoring DID methods that require unsustainably-sourced 
+        energy as part of their technical implementation or utilization should
+        be very carefully considered in balance against alternative approaches.
+      </p>
```

Comment:
```suggestion
      <p class="advisement">
        Utilizing or authoring DID methods that degrade security and 
        protection against the full range of attack vectors, interdiction 
        points, and centralization forces in trade for perceived gains in 
        system-external subjective factors should be very carefully 
        considered in balance against alternative approaches that 
        provide the highest levels of security and protection.
      </p>
```

Reply:
@csuwildcat how about something like this:

```
<p class="advisement">
  Utilizing or authoring DID methods that compromise on
  the principles of security and / or control for other principles
  of such as environmental concerns should be very carefully 
  considered in balance against alternative approaches that 
  provide the highest levels of security and protection
  depending on the use case involved.  For matters that concern
  human rights, selection of approaches that are weighted towards
  privacy and control should outweigh other considerations.
</p>

```

Thought being, that for areas where personal ID is involved I am highly skeptical of methods that do not opt for a proof of work approach at the current time, simply because of the privacy, security, and end user control compromises that are made with other approaches.  However, there are a lot of use cases for DIDs that may have lesser requirements in those areas.  A specific case I am thinking of is an area where our system issues a DID for discrete sources of data that have been identified.  We often utilize did:key for this, which then gives us options depending on the scenario and environment as to how to approach anchoring, etc. if required, but also gives us the option to lighten compute and network load where appropriate, while still being able to benefit from a DID based approach.

Reply:
The above is written in very strong subjective language (`degrade`, `perceived`, `subjective`, `highest`) which appears to be intended to be read as an objective analysis.  I wanted to suggest some language to help improve it, but it's as above -- switch from "try to conserve energy in your efforts to increase 'security'" to "consume all energy to increase 'security'!" -- to which I strongly object.

Reply:
@TallTed thoughts on the alternate language I offered?  I think it could use some edits

Reply:
@TallTed this is actually shifting the conversation to the more empirical topic of security, specifically protecting against "the full range of attack vectors, interdiction points, and centralization forces". These are critical aspects of system design that can be addressed with specific technical mitigations within implementations, thus it's probably a better basis for framing the considerations at bar.

I must push back strongly against the hyperbolic framing that this is switching to a premise that implementers should "consume all energy to increase 'security'!". There are many things wrong here, not least of which is that electricity != emissions (all electricity generation worldwide = just 25% of emissions), but the whole point of these changes is to avoid opening this up to inherently subjective value perceptions that the other premise will draw into the discussion.

If we assume a basis for articulation that invites subjective value perceptions, you may see PRs adding text that challenge people to justify why what may become some of the most important infra in human life is not valuable enough to protect with implementations that provide the highest levels of security. The most notable exemplar of a highly secure substrate commonly used today is equivalent to ~1/100th of the emissions impact from cow farts or 1/24th the emissions impact of clothes dryers. I personally believe what may become some of the most important infra for humans is worth ~1/100th of the emissions impact from cow farts or 1/24th the emissions impact of clothes dryers, but I don't want to have to ask others why they don't, which is why we should avoid that previous premise for articulating the considerations.

Reply:
> [@csuwildcat] the more empirical topic of security, specifically protecting against "the full range of attack vectors, interdiction points, and centralization forces".

At minimum, for this phrasing to do what you apparently meant, "full range" should be "full *known* range", as new vulnerability specimens surface regularly.

Next, please note that I referred nowhere to (presumably CO2 and other greenhouse gas) emissions in my words.  I was reacting to what you said earlier --

> `ultimately security and protection of critical systems, like identifier networks that may become the foundation for critical activities in human life, are the most important factors to account for.`

-- which I read as it was written -- i.e., security over all other considerations, amongst which I'm pretty sure you'll agree we find energy consumption.

Now, as to cow emissions... ["Over 95 percent, actually, is from the mouth, from the front end of the cow."](https://www.npr.org/sections/thesalt/2018/07/03/623645396/surf-and-turf-to-reduce-gas-emissions-from-cows-scientists-look-to-the-ocean) and [In a year, a single cow can belch around 220 pounds of methane](https://www.fastcompany.com/90658025/this-mask-for-cows-isnt-for-covid-it-helps-reduce-methane-emissions).  Those articles show a cute new mask for the cows that can reduce their belch emissions by \~98%, and they've also separately found that adding a bit of seaweed (\~1% or less, because of flavor issues -- not in the milk or beef, but the cows don't like >1%) to the cows' diet reduces overall emissions by ~80%, so your examples and/or figures will need some reworking.

Regarding clothes dryers, are you talking about gas or electric?  Or is this some magic aggregate?  Not that this is actually relevant to the discussion at hand, which, to my mind boils down to --

_Balance the concerns associated with the DID methods you're considering using or implementing, including their relative energy consumption at scale, their relative security, etc.  Sometimes higher security [note: there is no "high**est**" nor unbreakable security] is worth more energy consumption.  Sometimes lowering energy consumption is worth lower security.  It's horses for courses.  The right tool for the right job.  Don't use a hammer to drive a screw.  Don't use a saw to drive a nail._